general/administration/hardware-acceleration.md

```diff
@@ -43,10 +46,48 @@ The hardware acceleration is available immediately for media playback. No server
 
 Each hardware acceleration type, as well as each Jellyfin installation type, requires different setup options before it can be used. It is always best to consult the FFMpeg documentation on the acceleration type you choose for the latest information.
 
+### Acceleration on Docker
+
+In order to use Hardware acceleration in Docker, the devices must be passed to the container. To see what video devices are available, you can run `sudo lshw -c video` or `vainfo`
+
+Note: [NVIDIA GPU's](https://github.com/docker/compose/issues/6691) currently aren't supported in Docker-compose. 
+
+ Docker run configuration example:
+ 
+   `docker run -d \`  
```

Comment:
I would put this in block quotes like the other sections.

Reply:
I copied it from the installing page to make them match. I could fix both pages or leave it for now lol